Content.Server/Projectiles/ProjectileSystem.cs

```diff
@@ -43,12 +43,12 @@ private void OnStartCollide(EntityUid uid, ProjectileComponent component, ref St
             return;
         }
 
-        var ev = new ProjectileHitEvent(target);
+        var ev = new ProjectileHitEvent(component.Damage, target, component.Shooter);
         RaiseLocalEvent(uid, ref ev);
 
         var otherName = ToPrettyString(target);
         var direction = args.OurBody.LinearVelocity.Normalized();
-        var modifiedDamage = _damageableSystem.TryChangeDamage(target, component.Damage, component.IgnoreResistances, origin: component.Shooter);
+        var modifiedDamage = _damageableSystem.TryChangeDamage(target, ev.Damage, component.IgnoreResistances, origin: component.Shooter);
```

Comment:
Do you expect the damage value to have been changed by the raising of the event?

Reply:
huh?

Reply:
Oh uh, I think I know what you mean.
I'd like the damage value to be changed by the systems that are subscribed to that event, if that's what you mean.

Reply:
I'm pretty sure the event raising has no way to guarantee an order of operation. So *0.5 and -2 within two separate subscriptions will have an unpredictable outcome.

Maybe put whatever changes you want to do on the event's fields and guarantee the correct order of operations here?

Reply:
Ah I see what you mean.
I'll work on resolving that.

Reply:
> I'm pretty sure the event raising has no way to guarantee an order of operation. So *0.5 and -2 within two separate subscriptions will have an unpredictable outcome.

So I looked a bit into it and what I found is that people would use `DamageSpecifier.ApplyModifierSet` anyways, which shouldn't cause too many issues with the order at which stuff is done.